### PR TITLE
Review

### PR DESCRIPTION
**Pull request by AubidaNaalwa**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+}
+ 
+#websiteName{
+    font-size: 600%;
+    color: white;
+}
+#logoDiv{
+    background-image: url('personal_logo.jpeg');
+    background-repeat: no-repeat;
+
```
Comment (ameliawalcek) [inline style.css:24]: even though it is just the style.css try to make it look nice by remove spaces

File: controller.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
+
+const Controller = () => {
+    const moudl = moudle()
+    const vie = view()
+    const addingnewpost = (post) => {
```
Comment (ameliawalcek) [inline controller.js:5]: for arrow functions, if it has only one parameter you do not need to add `()`
also, you should be camelCasing you veriables in JS
`const addingNewPost = post => {`

File: controller.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
+
+const Controller = () => {
+    const moudl = moudle()
```
Comment (ameliawalcek) [inline controller.js:3]: there is no reason to shorten the name of your instances. The most important part of coding is clean code and to make sure people can easily read your code. By removing some letters, it only makes it difficult for someone to read. Shorter is not always better. 
`const moudle= Moudle()`

File: controller.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    const vie = view()
+    const addingnewpost = (post) => {
+        moudl.addPost(post)
+        vie.renderPosts(moudl.getUpdatedPosts())
+    }
+    const addingnewcomment = (postid, comment) => {
```
Comment (ameliawalcek) [inline controller.js:9]: `postID`
keep your variables consistent in your code

File: controller.js [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+        moudl.addAcommentToApost(postid, comment)
+        vie.renderPosts(moudl.getUpdatedPosts())
+    }
+    const deleteThePost = (id) => {
+        moudl.removeAPost(id)
+        vie.renderPosts(moudl.getUpdatedPosts())
+    }
+    const deleteTheComment = (postID, commentID) => {
+        moudl.removeACommentFromAPost(postID, commentID)
```
Comment (ameliawalcek) [inline controller.js:18]: you use `postID` here

File: controller.js [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+        vie.renderPosts(moudl.getUpdatedPosts())
+    }
+    const addNewUser = (user, pass) => {
```
Comment (ameliawalcek) [inline controller.js:21]: it is ok if your variables are a bit longer use the full word `password`

File: controller.js [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+        deleteTheComment,
+        addNewUser,
+        logInUser,
+        logout
+    }
+}
+const controller = Controller()
+$(`#grid-Twit`).on(`click`, `#twitbtn`, function () {
+
+    if (!$(`#text-input`).val())
```
Comment (ameliawalcek) [inline controller.js:52]: nice

File: controller.js (hunk 2 above)
Comment (AubidaNaalwa) [reply controller.js:5]: ok thanks .

File: controller.js [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+        deleteThePost,
+        deleteTheComment,
+        addNewUser,
+        logInUser,
+        logout
+    }
+}
+const controller = Controller()
+$(`#grid-Twit`).on(`click`, `#twitbtn`, function () {
+
```
Comment (ameliawalcek) [inline controller.js:51]: you use `$(`#text-input`).val()` twice. You should just define a variable up here
`const input = "$(`#text-input`).val()"`
then you do not need to repeat yourself

File: controller.js (hunk 3 above)
Comment (AubidaNaalwa) [reply controller.js:3]: no here I have a problem that returns me that I already define Moudle and it will not work same problem for the View

File: controller.js (hunk 4 above)
Comment (AubidaNaalwa) [reply controller.js:9]: ok I didnt saw that

File: controller.js (hunk 6 above)
Comment (AubidaNaalwa) [reply controller.js:21]: loool ooooch

File: controller.js [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+    let postcontainer = $(this).closest('.post-container')
+    controller.addingnewcomment(postcontainer.data().id, text.val())
+    text.val("")
+});
+
+
+
+$(`#grid-posts`).on(`click`, `.deleteBtn`, function () {
+    let postcontainer = $(this).closest('.post-container')
+    controller.deleteThePost(postcontainer.data().id)
```
Comment (ameliawalcek) [inline controller.js:73]: even though this is correct it is better to create new variables so someone can read your code easily. Instead, define a variable `id`.


`const id = postcontainer.data().id`
`let postcontainer = $(this).closest('.post-container')`
`controller.deleteThePost(id)`

File: controller.js [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+
+});
+
+
+$(`#grid-posts`).on(`click`, `.X`, function () {
+    let postcontainer = $(this).closest('li')
+    controller.deleteTheComment($(this).closest('.post-container').data().id, postcontainer.data().id)
```
Comment (ameliawalcek) [inline controller.js:80]: when you are sending arguments into a function make sure you send them as veriables. By having long lines of code it is hard to follow. Clean and simple code is best. 

`const postID= $(this).closest('.post-container').data().id`
`const commentID= postcontainer.data().id`
`controller.deleteTheComment(postID, commentID)`

File: controller.js [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+    $(`#container`).css('display', 'grid')
+    $(`#loginForm`).css('display', 'none')
+});
+
+$(`#signup`).on(`click`, function () {
+    if (!$(`#userName`).val() || !$(`#password`).val()) {
+        alert(`missing Values`)
+        return
+    }
+    controller.addNewUser($(`#userName`).val(), $(`#password`).val())
```
Comment (ameliawalcek) [inline controller.js:94]: define the arguments before putting them into the function like I showed above

File: moudle.js [hunk 12]
```diff
@@ -0,0 +1,94 @@
+const moudle = () => {
+    let posts = []
+    let postId = 0;
+    let commentID = 0;
+    let userName = []
```
Comment (ameliawalcek) [inline moudle.js:5]: for something like this you can create an object
`cosnt user = {name: '', id: 0, currentUser: -1}`

File: controller.js (hunk 8 above)
Comment (AubidaNaalwa) [reply controller.js:51]: you are correct

File: view.js [hunk 13]
```diff
@@ -0,0 +1,44 @@
...
+            <div class="replyContainer">
+                <input type="text" name="inputText" class="commentText" placeholder ="Got a something to say?">
+                <button class ="commentBtn">Comment</button>
+                <button class ="deleteBtn">delete Post</button>
+            </div>
+        </div>`;
+            $("#grid-posts").prepend(postContainer);
+            commentContainer = $(`#post-${post.id}`).find(".commentContainer")
+            for(let comment of post.comments){
+                commentContainer.append(`<li class ="comment-${comment.id}" data-id = ${comment.id}><span class ="X fas fa-trash">  </span><span>${comment.comment}</span></li>`)
```
Comment (ameliawalcek) [inline view.js:23]: define a veriable of what you are appending instead of putting long lines of code in the `.append()` method.

**Review by ameliawalcek — COMMENTED** [review]
Very nice job! really clean code. 

I gave you some comments. Look them over and let me know if you have any questions.
